src/org/traccar/protocol/OsmAndProtocolDecoder.java

```diff
@@ -42,8 +42,7 @@ public OsmAndProtocolDecoder(OsmAndProtocol protocol) {
     }
 
     @Override
-    protected Object decode(
-            Channel channel, SocketAddress remoteAddress, Object msg) throws Exception {
+    protected Object decode(Channel channel, SocketAddress remoteAddress, Object msg) throws Exception {
```

Comment:
Why are you changing OsmAnd protocol?

Reply:
I looked into this file for some how to. But it looks like my environment tried to change the formatting, can be undone.